BITS-App/App.xaml.cs

```diff
@@ -9,8 +9,10 @@ public partial class App : Application {
 	public App() {
 		InitializeComponent();
 
-		client = new HttpClient();
+        MainPage = new NavigationPage(new Pages.HomePage());
```

Comment:
Ah yes, we set MainPage to a NavigationPage and then proceed to throw it away _immediately_